spec/models/order_spec.rb

```diff
@@ -13,4 +13,8 @@
     expect(Order.first.user_id).to eq(user.id)
   end
 
+  it "has an cart" do
+    expect(order.cart).to eq({"1"=>"3", "2"=>"3", "3"=>"3"})
```

Comment:
Redundant curly braces around a hash parameter.<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.